src/Types/ConsensusInfo.cs

```diff
@@ -129,12 +136,15 @@ public sealed record ConsensusInfo(
     ulong FinalizationCount,
     DateTimeOffset? LastFinalizedTime,
     double? FinalizationPeriodEma,
-    double? FinalizationPeriodEmsd
+    double? FinalizationPeriodEmsd,
+    ConcordiumBftDetails? ConcordiumBftDetails
     )
 {
-    internal static ConsensusInfo From(Grpc.V2.ConsensusInfo consensusInfo) =>
-        new
-        (
+    internal static ConsensusInfo From(Grpc.V2.ConsensusInfo consensusInfo)
+    {
+        var _ = ConcordiumBftDetails.TryFrom(consensusInfo, out var concordiumBftDetails);
```

Comment:
Why do we implement the try method, if we discard the return value and pass concordiumBftDetails regardless of its value?